### PR TITLE
docs: tidy up noise reduction essay

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

This fixes some small issues.
I'm not making huge changes here, because I want to preserve the style of the original author.

## Context:

Helps with issue #7626.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/noise-reduction.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ description: How to reduce the "noise" associated with module updates
...
 Indeed, if you leave Renovate on its default settings of raising a PR every single time any dependency receives any update.. you will get a lot of PRs and related notifications.
-This document will give you some ideas of how to reduce the amount of "noise" in your repository and the Pros/Cons of each.
+This document will give you some ideas of how to reduce the amount of "noise" in your repository and the Pros/Cons of each approach.
 
 Of course, please keep in mind that people's definitions of "noise" may differ.
 For some people, it's noisy only if they get a notification or email from GitHub.
 For others, too many commits in their `master` branch may be "noise".
 In other words, your mileage may vary.
-Please contact the author if you have any ideas on this topic!
+Please contact the author if you have any ideas on this topic! (How to contact the author (Rhys), do you want them to mail, Twitter, open a issue on GitHub?)
```
Comment (HonkingGoose) [inline docs/usage/noise-reduction.md:17]: How do we want readers to contact the author?
Also we don't explicitly say who is the author of this document?

I think @rarkins wrote this?

Comment (viceice) [reply docs/usage/noise-reduction.md:17]: I think best ways is to open an issue to contact the maintainer 🙃
